### ISSUE TITLE
Why is deepspeed enabled in the Bloom training script?

### ISSUE THREAD

**Issue by robertLiuLinFeng** [bigscience-workshop/bigscience#71]
Why is the value of Zero-State 0 when deepspeed is enabled in the Bloom training script? Can the Bloom model be trained and the loss curve is aligned when deepspeed is disabled? Thanks very much.

```shell
DEEPSPEED_ARGS=" \
    --deepspeed \
    --deepspeed_config ${config_json} \
    --zero-stage ${ZERO_STAGE} \
    --deepspeed-activation-checkpointing \
    "
```
